test/infamy/container.py

```diff
@@ -17,6 +19,12 @@ def _find(self, name):
                 return container
         return None
 
+    def _warn(self, msg):
+        """Print a warning message in yellow to stderr."""
+        GUL = "\033[93m"
+        RST = "\033[0m"
+        print(f"{GUL}warn - {msg}{RST}")
+
```

Comment:
I think this should be a general utility function, lest we have 7 variants of this one in a year's time.

Reply:
You found my green goblin, now where should it live?

Reply:
`test/infamy/util.py` maybe?